### DISCUSSION TITLE
2.5.5 on Mac M1

### DISCUSSION THREAD

**Discussion by BlackAlbatros** [upscayl/upscayl#385] [Releases]
Hi.
There are serious issues with this version on Mac M1.
Most of the time the Upscayl button doesn't respond or it get's stuck picking a model.
Any ideas?

**Comment by aaronliu0130** [comment]
For example? Could you send a video? Also, this should be an issue.

**Comment by Ericmeister** [comment]
> Hi. There are serious issues with this version on Mac M1. Most of the time the Upscayl button doesn't respond or it get's stuck picking a model. Any ideas?

I couldn't open it at first as well. I had to authorise it to start in my security settings since it is from an unkown developer. After that it worked great on 16 inch Mac Pro M1 Max 32gb Ram 24 Core Gpu
